### DISCUSSION TITLE
alist无法预览图片，使用ip地址可以

### DISCUSSION THREAD

**Discussion by xiaocaoyf** [AlistGo/alist#6867] [General]
使用域名访问alist，存储里面的图片可以下载，但是不能预览。
如果使用ip地址访问alist，可以正常显示预览。
使用nginx
![微信截图_20240724144603](https://github.com/user-attachments/assets/f90b9414-3939-4b6b-841f-6d31b9bb43c1)
![微信截图_20240724144759](https://github.com/user-attachments/assets/60a91a84-cd05-49d9-bb8c-305a07896a87)

**Comment by anwen-anyi** [comment]
把配置文件里面的site url配置一下试试呢
